main.js

```diff
@@ -0,0 +1,144 @@
+        brand: 'Meinl',
+        price: 129.99,
+        description: 'A frame drum for people who like to jig.',
+        details: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.',
+        origin: 'Ireland',
+        imageUrl: 'https://goo.gl/YNb74O'
+      },
+      {
+        itemId: 8,
+        name: 'TR-09',
+        brand: 'Roland',
+        price: 399,
+        description: 'A drum machine for people with no desk space.',
+        details: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.',
+        origin: 'Japan',
+        imageUrl: 'https://goo.gl/gJYUxz'
+      }
+    ]
+  },
+  details: {
+    item: null
+  }
+}
+
+function renderCatalogItem(catalog) {
+  var $card = document.createElement('div')
+  $card.classList.add('card')
+  $card.setAttribute('style', 'width: 18rem')
+
+  var $cardImg = document.createElement('img')
+  $cardImg.classList.add('card-imt-top')
+  $cardImg.setAttribute('src', catalog.imageUrl)
+  $card.appendChild($cardImg)
+
+  var $cardBody = document.createElement('div')
+  $cardBody.classList.add('card-body')
+  var $cardBodyHeader = document.createElement('h5')
+  $cardBodyHeader.classList.add('card-title')
+  $cardBodyHeader.textContent = catalog.name
+  var $cardBodyPrice = document.createElement('p')
+  $cardBodyPrice.classList.add('card-text')
+  $cardBodyPrice.textContent = '$' + catalog.price
+
+  $cardBody.appendChild($cardBodyHeader)
+  $cardBody.appendChild($cardBodyPrice)
+
+  $card.appendChild($cardBody)
+
+  return $card
+}
+
+function renderGridCatalog(catalog) {
+  var $gridContainer = document.createElement('div')
+  $gridContainer.classList.add('container')
+
+  var $gridHeader = document.createElement('h1')
+  $gridHeader.textContent = 'Jamazon'
+  $gridContainer.appendChild($gridHeader)
+
+  var $gridRow = document.createElement('div')
+  $gridRow.classList.add('row')
+
+  catalog.catalog.items.forEach(function (object, i) {
+    let $cardObject = renderCatalogItem(catalog.catalog.items[i])
+    $gridRow.appendChild($cardObject)
+  })
+
+  $gridContainer.appendChild($gridRow)
+
+  return $gridContainer
+}
+
+function appendGridCatalog(catalog) {
+  document.body.appendChild(renderGridCatalog(catalog))
```

Comment:
You should insert this into the `<div data-view="catalog"></div>` instead of the `body`.